src/fiesta/inference/prior/prior.py

```
@@ -235,6 +235,55 @@ def log_prob(self, x: dict[str, Array]) -> Float:
+        **kwargs,
+    ):
+        super().__init__(naming, transforms)
+        assert self.n_dim == 1, "TruncatedNormal needs to be 1D distributions"
+        self.mu = mu
+        self.sigma = sigma
+        self.xmin = xmin
+        self.xmax = xmax
+
+    def sample(
+        self, rng_key: PRNGKeyArray, n_samples: int
+    ) -> dict[str, Float[Array, " n_samples"]]:
```

Ruff (0.15.6)

[error] Syntax error in forward annotation: Expected an expression

(F722)